### PR TITLE
[AutoBuild] Add revision on Yggdrasil to the registration PR

### PR DESCRIPTION
**Pull request by giordano**
Ref https://github.com/JuliaPackaging/Yggdrasil/issues/2433.  CC: @DilumAluthge.

### CODE REVIEW

File: src/AutoBuild.jl [hunk 1]
```diff
@@ -2,7 +2,7 @@ export build_tarballs, autobuild, print_artifacts_toml, build, get_meta_json
 import GitHub: gh_get_json, DEFAULT_API
 import SHA: sha256, sha1
 using Pkg.TOML, Dates, UUIDs
-using RegistryTools, Registrator
+using RegistryTools
```
Comment (giordano) [inline src/AutoBuild.jl:5]: @DilumAluthge can you please confirm `Registrator` doesn't export any symbol?  If not, I don't think we use it here anywhere.

Comment (DilumAluthge) [reply src/AutoBuild.jl:5]: Registrator doesn't export anything. But you do use a Registrator function in src/wizard/GitHub.jl.

Comment (giordano) [reply src/AutoBuild.jl:5]: I know, but that's a different module :slightly_smiling_face: 
https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/fe019a6f4e1c5a218be0be7dce8df6601d2e3810/src/wizard/github.jl#L1

**Comment by giordano** [comment]
```
[ Info: Building for x86_64-linux-gnu-cxx03
[ Info: Directory "/data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/build_tests" found
[ Info: Building for x86_64-linux-gnu-cxx03
[ Info: Directory "/data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/build_tests" found
docker: Error response from daemon: rename /var/lib/docker/containers/81101ecade5b2c7d417a44a1c4ff3480f39cf098df273003f3858497c67bf8b6/hash986915928 /var/lib/docker/containers/81101ecade5b2c7d417a44a1c4ff3480f39cf098df273003f3858497c67bf8b6/resolv.conf.hash: no space left on device.
Auditor - cxxabi selection: Test Failed at /data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/auditing.jl:233
  Expression: detected_cxxstring_abi == cxxstring_abi(platform)
   Evaluated: nothing == "cxx03"
Stacktrace:
 [1] (::var"#64#70"{Platform})(oh::ELFHandle{IOStream})
   @ Main /data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/auditing.jl:233
 [2] readmeta(f::var"#64#70"{Platform}, file::String)
   @ ObjectFile /data/staticfloat/bb_azp/rootfs/agent_3/depot/packages/ObjectFile/QqgZQ/src/Abstract/ObjectHandle.jl:158
 [3] (::var"#63#69"{var"#do_build#68"{LibraryProduct}, LibraryProduct, Platform, VersionNumber})(build_path::String)
   @ Main /data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/auditing.jl:231
 [4] mktempdir(fn::var"#63#69"{var"#do_build#68"{LibraryProduct}, LibraryProduct, Platform, VersionNumber}, parent::String; prefix::String)
   @ Base.Filesystem ./file.jl:711
 [5] mktempdir(fn::Function, parent::String)
   @ Base.Filesystem ./file.jl:709
 [6] top-level scope
   @ /data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/auditing.jl:211
 [7] top-level scope
   @ /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144
 [8] top-level scope
   @ /data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/auditing.jl:185
[ Info: Building for x86_64-linux-gnu-cxx03
[ Info: Directory "/data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/build_tests" found
[ Info: Building for x86_64-linux-gnu-cxx11
[ Info: Directory "/data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/build_tests" found
[ Info: Building for x86_64-linux-gnu-cxx11
[ Info: Directory "/data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/build_tests" found
[ Info: Building for x86_64-linux-gnu-cxx11
[ Info: Directory "/data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/build_tests" found
Test Summary:              | Pass  Fail  Total
Auditor - cxxabi selection |   28     1     29
ERROR: LoadError: LoadError: Some tests did not pass: 28 passed, 1 failed, 0 errored, 0 broken.
in expression starting at /data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/auditing.jl:184
in expression starting at /data/staticfloat/bb_azp/rootfs/agent_3/_work/1/s/test/runtests.jl:63
```
[what the...](https://dev.azure.com/JuliaPackaging/BinaryBuilder.jl/_build/results?buildId=8601&view=logs&j=54759826-d195-5ca2-1571-21806663cd36&t=9eb882d6-5a34-5e37-1506-d0b4bf16214f&l=752)

**Pull request merged**